src/schema/objects/extensions.yaml

```diff
@@ -16,6 +16,15 @@ bdf:
     Each recording consists of a single `.bdf` file.
     [`bdf+`](https://www.teuniz.net/edfbrowser/bdfplus%20format%20description.html) files are permitted.
     The capital `.BDF` extension MUST NOT be used.
+dtseriesnii:
+  value: .dtseries.nii
+  display_name: CIFTI-2 Dense Time Series
+  description: |
+    This file type represents data points in a series for every vertex and voxel in the mapping. A
+    row is a complete data series, for a single vertex or voxel in the mapping that applies along the
+    second dimension. A dataseries is often a timeseries, but it can also represent other data types
+    such as a series of sampling depths along the surface normal from the white to pial surface. It
+    retains the ‘t’ in dtseries from CIFTI-1 naming conventions.
```

Comment:
```suggestion
  display_name: CIFTI-2 Dense Data Series
  description: |
    A CIFTI-2 dense data series file.
    
    A data series file represents data points in a series for every vertex and voxel in the mapping. A
    row is a complete data series, for a single vertex or voxel in the mapping that applies along the
    second dimension. A data series is often a time series, but it can also represent other data types
    such as a series of sampling depths along the surface normal from the white to pial surface.
```